src/cmd.ts

```diff
@@ -11,6 +11,15 @@ export async function run_yarn(targetDir: string): Promise<void> {
   }
 }
 
+export function git_branch_name(): string {
+  try {
+    let branch = execSync("git branch --show-current")
+    return branch.toString()
```

Comment:
```suggestion
    return execSync("git branch --show-current", { encoding: "utf8" })
```
Just like with `readFile` it's better to be explicit about encoding and get a string back instead of converting a buffer using an implicit encoding.